jekyll/_cci2/use-the-cli-and-vcs-for-config-policy-management.adoc

```diff
@@ -15,29 +15,34 @@ NOTE: Config policy management is available on the **Scale** plan and is current
 
 CAUTION: While config policy management is in open preview stage, it should **not** be used for compliance purposes. The feature might become unavailable occasionally during **open preview**, during which build configurations may **not** be evaluated against policies.
 
-CAUTION: Ensure you have authenticated your version of the CLI with a token, and updated the CLI, before attempting to use the CLI for config policy management. See the link:/docs/local-cli[Installing the Local CLI] page for more information.
+Follow the how-to guides on this page to learn how to perform various config policy management processes.
 
-Use the CircleCI CLI to manage your organization's policies programmatically. The sub-commands to perform policy management are grouped under the `policy` command. 
+[#config-policy-management-enablement]
+== Enable or disable policy evaluation for an organization
```

Comment:
Here I've just copied info on enabling the feature to the main feature page, from one of the how to guides